libnymea-core/scriptengine/scriptaction.cpp

```diff
@@ -156,7 +156,10 @@ void ScriptAction::execute(const QVariantMap &params)
         }
         action.setParams(paramList);
         qCDebug(dcScriptEngine()) << "Executing action:" << action.thingId() << action.actionTypeId() << action.params();
-        m_thingManager->executeAction(action);
+        ThingActionInfo *actionInfo = m_thingManager->executeAction(action);
+        connect(actionInfo, &ThingActionInfo::finished, this, [](){
```

Comment:
empty lambda

Reply:
whoops. forgot to log the action in here (only added it to a ThingState::setValue(), Added now.